### DISCUSSION TITLE
values parsing error

### DISCUSSION THREAD

**Discussion by baiyibing123** [vectordotdev/helm-charts#356] [Q&A]
<img width="528" alt="image" src="https://github.com/vectordotdev/helm-charts/assets/14048464/a42a5ec3-e9ff-4247-80b6-b5e6067843ee">
<img width="1752" alt="image" src="https://github.com/vectordotdev/helm-charts/assets/14048464/d6a49de9-84bf-4dac-9da7-ee4aee71b250">

**Comment by baiyibing123** [comment]
The escape cannot take effect

**Reply by jszwedko** [reply]
You'll need to escape the Helm templating like:

```
project: {{ "{{" }} fields.project {{ "}}" }}
```

**Reply by baiyibing123** [reply]
I tried to report an error
Error: failed to parse values.yaml: error converting YAML to JSON: yaml: line 89: did not find expected key

That's OK
project: '{{ "{{ fields.project }}" }}'
